src/emm/services/modules_service.py

```diff
@@ -231,3 +231,45 @@ def git_commit_modules(self, commit: str) -> None:
             module_data = self.get_module_data(module)
             module_location = Path(module_data.prefix) / module
             self.git_service.commit_all(commit, module_location)
+
+    def add_pr_link(self, comments: Dict[str, str]) -> None:
+        """
+        Add link to each module.
+
+        :param comments: Dictionary of module and its pull request url.
+        """
+        for module, pr_url in comments.items():
+            if module != "base":
+                module_data = self.get_module_data(module)
+                module_location = Path(module_data.prefix) / module
+                pr_links = "\n".join(
```

Comment:
This message should probably be a little more descriptive in describing what it is. Right now, it looks like it will just show up as a bunch of random links. It should probably explain what all the links are.

Reply:
This message would look like:
module_1 pr: "link/to/module_1_repo/pull/123"
module_2 pr: "link/to/module_2_repo/pull/234"
 
Do we want to add additional information other than that? I think this info is straightforward, because the link to pr is a hyperlink. User can click and redirect to that pull request, it also contains module name ahead of the link.